main/getting-started/intro-hackathon.md

```diff
@@ -0,0 +1,103 @@
+  
+- [Building a Composable DeFi Contract](https://www.youtube.com/watch?v=faxrecQgEio) by Kate Sills walks through programming first a covered call option contract,
+and then writing an OTC Desk contract that uses the covered call contract.
+
+## ERTP and Zoe Guides and APIs
+
+For a deeper explanation of Agoric concepts, what you can accomplish with a contract,
+and what Agoric API commands do, read the following:
+
+- ERTP: The **[ERTP Guide](/ertp/guide/README.md)** is a detailed description of 
+  ERTP concepts, design, components, and commands. The ERTP API is documented
+  [here](/ertp/api/#ertp-api). 
+- Zoe: The **[Zoe Guide](/zoe/guide/README.md)** is a detailed description of 
+  Zoe concepts, design, components, and commands. The Zoe API is documented
+  [here](/zoe/api/#zoe-api).
+  
+## Example contracts
+
+To familiarize yourself with working Agoric smart contracts, take a look at our 
+[sample contracts](/zoe/guide/contracts/README.md). They are:
+- [Atomic Swap](/zoe/guide/contracts/atomic-swap.md)
+- [Autoswap](/zoe/guide/contracts/autoswap.md)
+- [Barter Exchange](/zoe/guide/contracts/barter-exchange.md)
+- [Covered Call](/zoe/guide/contracts/covered-call.md)
+- [Multipool Autoswap](/zoe/guide/contracts/multipoolAutoswap.md)
+- [Second-price auction](/zoe/guide/contracts/second-price-auction.md)
+- [Simple Exchange](/zoe/guide/contracts/simple-exchange.md)
+
+## Advanced or specialized documentation
+
+As you start writing contracts, you should read our
+**[JavaScript Distributed Programming Guide](/distributed-programming.md)**. 
+We've made some Agoric-specific additions at various layers, including concepts, syntax, 
+and additions to the Agoric library. You should know about and understand these before 
+doing significant programming on the Agoric platform.
+
+More specialized documentation includes:
+- **[Wallet API Reference](/wallet-api.md)**: How to work with an Agoric wallet, which stores users' assets.
+- **[Agoric CLI Guide](/getting-started/agoric-cli-guide.md)**: Details about Agoric CLI commands, which
+  install dependencies, initialize, deploy, and start Agoric projects. It also tells how to use 
+  the Agoric CLI to develop and test multiuser Dapps by deploying and starting a local-chain-multiuser scenario.
+- **[Deploying Smart Contracts](./deploying.md)**: A brief description of the tools and processes 
+  for deploying contracts to the chain and application code to the application server.
+- **[dIBC (Dynamic Inter-Blockchain Communication Protocol) Guide](https://github.com/Agoric/agoric-sdk/blob/master/packages/SwingSet/docs/networking.md)**:
+  Our enhancement to [IBC](https://cosmos.network/ibc) for connecting to services on other blockchains or making
+  services on the Agoric blockchain available to other blockchains. 
+
+Finally, our ultimate documentation is our **[GitHub
+repository](https://github.com/Agoric/)** for the code that defines the Agoric SDK.
```

Comment:
If the link text says repo, let's actually link to that, not the Agoric organization 
```suggestion
repository](https://github.com/Agoric/agoric-sdk)** for the code that defines the Agoric SDK.
```